Add task doc for chef (#6192)
grafana/agent: docs/sources/flow/get-started/install/chef.md

---
aliases:
- /docs/grafana-cloud/agent/flow/get-started/install/chef/
canonical: https://grafana.com/docs/agent/latest/flow/get-started/install/chef/
description: Learn how to install Grafana Agent Flow with Chef
menuTitle: Chef
title: Install Grafana Agent Flow with Chef
weight: 550
---

> # Install or uninstall {{% param "PRODUCT_NAME" %}} using Chef

Reviewer: ```suggestion
# Install {{% param "PRODUCT_NAME" %}} with Chef
```

You can use Chef to install and manage {{< param "PRODUCT_NAME" >}}.

## Before you begin

- These steps assume you already have a working [Chef](https://www.chef.io/products/chef-infrastructure-management/) setup.

> - You can add the resources below to any new or existing recipy you choose.

Reviewer 2: ```suggestion
- You can add the following resources to any new or existing recipe you choose.
```

> - These tasks install {{< param "PRODUCT_NAME" >}} from the package repositories. They expect to target Linux systems from the following families:

Reviewer 2: ```suggestion
- These tasks install {{< param "PRODUCT_NAME" >}} from the package repositories. The tasks target Linux systems from the following families:
```

  - Debian (including Ubuntu)
  - RedHat Enterprise Linux
  - Amazon Linux
  - Fedora

## Steps

To add {{% param "PRODUCT_NAME" %}} to a host:

> 1. Add the following resources to a recipe to add the Grafana package repositories to your system:

Reviewer: ```suggestion
1. Add the following resources to your [Chef][] recipe to add the Grafana package repositories to your system:
```
